Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -1821,22 +1822,18 @@ WI._saveCookieForOpenTabs = function()
     }
 };
 
-WI._windowFocused = function(event)
+WI._updateWindowInactiveState = function(event)
 {
-    if (event.target.document.nodeType !== Node.DOCUMENT_NODE)
-        return;
-
     // FIXME: We should use the :window-inactive pseudo class once https://webkit.org/b/38927 is fixed.
```

Comment:
Out of curiosity, would `:window-inactive` (if it was fully implemented) work for this instead?

Reply:
Yes, see the FIXME that is there.